.github/workflows/publish.yaml

```
@@ -0,0 +1,68 @@
+name: Publish App
```

```suggestion
name: App Publish
```

Also, file name should be "app-publish.yaml"